fastlane/report.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
We don't commit these files, check the `.gitignore` in another repo and copy the fastlane lines over